k8s/upgrade/src/constant.rs

```diff
@@ -29,7 +35,8 @@ pub(crate) const UPGRADE_BINARY_NAME: &str = "upgrade-job";
 /// Upgrade job container name.
 pub(crate) const UPGRADE_JOB_CONTAINER_NAME: &str = "mayastor-upgrade-job";
 /// Upgrade job container image.
-pub(crate) const UPGRADE_JOB_IMAGE: &str = "openebs/mayastor-upgrade-job:develop";
+pub(crate) const UPGRADE_JOB_IMAGE: &str =
+    formatcp!("{:?}:{:?}", UPGRADE_JOB_IMAGE_REPO, UPGRADE_JOB_IMAGE_TAG);
```

Comment:
you can also simply replace upgrade_job_image with a function and avoid adding dependency just for string concat:
```rust
fn upgrade_job_image() -> String {
  format!("{:?}:{:?}", UPGRADE_JOB_IMAGE_REPO, UPGRADE_JOB_IMAGE_TAG)
}
```
and you already do something similar with `upgrade_name_concat`.